src/app/pages/home-page/components/technologies-section/technologies-section.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+  <div class="text-container">
+    <h1>Technologies</h1>
+    <p>The are some of the technologies we focus on at GDSC UPT</p>
+  </div>
+  <mat-grid-list [cols]="breakpoint" rowHeight="2:1" (window:resize)="onResize($event)"
+                 [gutterSize]="'30px'">
```

Comment:
please use this type of code formatting:
```suggestion
  <mat-grid-list
            [cols]="breakpoint"
            rowHeight="2:1" 
            (window:resize)="onResize($event)"
            [gutterSize]="'30px'"
 >
```